examples/generic/ros/can_bridge/project.xml

```diff
@@ -2,6 +2,7 @@
   <extends>modm:blue-pill</extends>
   <options>
     <option name="modm:build:build.path">../../../../build/generic/ros/can_bridge</option>
+    <option name="modm:build:openocd.cfg">openocd.cfg</option>
```

Comment:
You still need to actually add an `openocd.cfg` file. See https://github.com/modm-io/modm/pull/240/files for the recommended content.

Reply:
Just copied from the blue pill example 😂